main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
+from django.utils import timezone
+import datetime
+
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data.get("race", {})
+        if race_data:
+            race_name = race_data.get("name")
+            race_description = race_data.get("description", "")
+            race, created = Race.objects.get_or_create(
+                name=race_name, defaults={"description": race_description}
+            )
```

Comment:
avoid creating additional variables and instead pass the arguments directly within the `get_or_create`

Reply:
Fix in all places